test/regression/gufi_stat.expected

```diff
@@ -45,6 +47,13 @@ unusual#? directory
 $ gufi_stat --format '%N %n %a %A %d %D %f %F %m %s %w %W %x %X %y %Y\n' "prefix/.hidden"
 'prefix/.hidden' prefix/.hidden 664 -rw-rw-r-- ? ? 81b4 regular file   9 - 0 1970-01-01 00:00:09 +0000 9 1970-01-01 00:00:09 +0000 9
 
+$ gufi_stat --verbose --format '%N %n %a %A %d %D %f %F %m %s %w %W %x %X %y %Y\n' "prefix/.hidden"
+'prefix/.hidden' prefix/.hidden 664 -rw-rw-r-- ? ? 81b4 regular file   9 - 0 1970-01-01 00:00:09 +0000 9 1970-01-01 00:00:09 +0000 9
+GUFI query is
+   gufi_stat_bin
+    -f %N %n %a %A %d %D %f %F %m %s %w %W %x %X %y %Y\n
+    prefix/.hidden
+
```

Comment:
The result is printed after the verbose output.

Reply:
Weird. I'll take a look.

Reply:
I think all of them are printing the results before the verbose output first for some reason.

Reply:
So weird. I just added sys.stdout.flush() to gufi_common.py and now suddenly a bunch of tests are failing. I'm looking into it but I'm not sure what's causing these errors.